packages/codemirror-extension/src/commands.ts

```diff
@@ -103,6 +105,21 @@ export const commandsPlugin: JupyterFrontEndPlugin<void> = {
       isEnabled
     });
 
+    app.commands.addCommand(CommandIDs.toggleTabFocusMode, {
+      label: trans.__('Toggle Tab Focus Mode'),
+      caption: trans.__(
+        'Toggles behaviour of Tab key between inserting indentation and moving to next focusable element'
```

Comment:
Looks like most (but not all) of the code base follows the other spelling, so maybe better to keep it consistent (also w.r.t localization)?

```suggestion
        'Toggles behavior of Tab key between inserting indentation and moving to next focusable element'
```